content/en/docs/aws/pipeline.md

```diff
@@ -6,7 +6,7 @@ weight = 90
 
 ## Authenticate Kubeflow Pipeline using SDK inside cluster
 
-In v1.1.0, in-cluster communitation from notebook to Kubeflow Pipeline is not supported in this phase. In order to use `kfp` as previous, user needs to pass a cookie to KFP for communication as a walkaround.
+In v1.1.0, in-cluster communication from notebook to Kubeflow Pipeline is not supported in this phase. In order to use `kfp` as previous, user needs to pass a cookie to KFP for communication as a walkaround.
```

Comment:
```
 user needs to pass a cookie to KFP for communication as a walkaround
```

Seems like `walkaround` should be `work-around`. Can you also adopt this change?